mlflow/tracing/archival/databricks.py

```diff
@@ -0,0 +1,405 @@
+from mlflow.utils.databricks_utils import get_databricks_host_creds
+from mlflow.utils.mlflow_tags import MLFLOW_EXPERIMENT_TRACE_ARCHIVAL_TABLE
+from mlflow.utils.rest_utils import http_request
+from mlflow.utils._spark_utils import _get_active_spark_session
+from google.protobuf.json_format import MessageToDict
+
+_logger = logging.getLogger(__name__)
+
+# Supported schema version for trace archival
+SUPPORTED_SCHEMA_VERSION = "v1"
+
+
+def _validate_schema_versions(spans_version: str, events_version: str) -> None:
+    """
+    Validate that both spans and events tables use supported schema versions.
+    
+    Args:
+        spans_version: Schema version of the spans table
+        events_version: Schema version of the events table
+        
+    Raises:
+        MlflowException: If either table uses an unsupported schema version
+    """        
+    if spans_version != SUPPORTED_SCHEMA_VERSION:
+        raise MlflowException(
+            f"Unsupported spans table schema version: {spans_version}. "
+            f"Only {SUPPORTED_SCHEMA_VERSION} is supported for GenAI trace views."
+        )
+    
+    if events_version != SUPPORTED_SCHEMA_VERSION:
+        raise MlflowException(
+            f"Unsupported events table schema version: {events_version}. "
+            f"Only {SUPPORTED_SCHEMA_VERSION} is supported for GenAI trace views."
+        )
+    
+    _logger.debug(f"Schema version validation passed: spans={spans_version}, events={events_version}")
+
+
+def _create_genai_trace_view(view_name: str, spans_table: str, events_table: str) -> None:
+    """
+    Create a logical view for GenAI trace data that combines spans and events tables.
+    
+    Args:
+        view_name: The name of the final view to create (e.g., 'catalog.schema.trace_logs_12345')
+        spans_table: The name of the table containing raw spans data
+        events_table: The name of the table containing raw events data
+        
+    Raises:
+        MlflowException: If view creation fails
+    """
+    try:
+        spark = _get_active_spark_session()
+        if spark is None:
+            from pyspark.sql import SparkSession
+            spark = SparkSession.builder.getOrCreate()
+            
+        query = f"""
+            CREATE OR REPLACE VIEW {view_name} AS
+            WITH root_spans AS ( -- 1. Filter and extract attributes efficiently
+              SELECT
+                trace_id,
+                -- Extract all needed attributes in single pass
+                attributes['mlflow.traceRequestId'] AS client_request_id,
+                attributes['mlflow.spanInputs'] AS request,
+                attributes['mlflow.spanOutputs'] AS response,
+                attributes['mlflow.experimentId'] AS experiment_id,
+                attributes['inference.table_name'] AS inference_table_name,
+                attributes AS trace_metadata,
+                TIMESTAMP(start_time_unix_nano / 1000000000) AS request_time,
```

Comment:
```suggestion
                TIMESTAMP(start_time_unix_nano / 1000000) AS request_time,
```

`request_time` in `TraceInfo` is millisecond. Since `execution_duration_ms` is computed correctly below, I guess this is typo?